eZ/Bundle/EzPublishCoreBundle/Resources/config/fos_http_cache.yml

```diff
@@ -7,3 +7,5 @@ user_context:
     enabled: true
     hash_cache_ttl: 600
     user_hash_header: X-User-Hash
+    user_identifier_headers:
+        - Authorization
```

Comment:
this is assuming your app only uses basic auth and sessions tracked with a cookie